code/controllers/subsystem/camera.dm

```diff
@@ -0,0 +1,34 @@
+// Subsystem that controls the camera vision tiles
+SUBSYSTEM_DEF(camera)
+	name = "Camera"
+	flags = SS_BACKGROUND | SS_NO_INIT
+	priority = FIRE_PRIORITY_AMBIENCE
```

Comment:
```suggestion
	priority = FIRE_PRIORITY_CAMERA
```
Also might wanna set an SS_impact var or whatever I named it

Reply:
Oh whoops. I made it but never used it I see

Reply:
`cpu_display` you mean? It defaults to `SS_CPUDISPLAY_DEFAULT` which I think is appropriate since each update can be quite expensive but they don't happen too often

Reply:
*shrug*